structs/structs.go

```diff
@@ -34,14 +34,14 @@ type Course struct {
 	CourseTitle       string     `json:"courseTitle"`
 	RequiredEquipment []string   `json:"requiredEquipment,omitempty"`
```

Comment:
Maybe this can be deleted as it's not part of anything iirc anymore

Reply:
Yeah I was thinking the same thing as we haven't ever used it

Reply:
Removed them